src/component/common/Table/TableSearch/TableSearchField/TableSearchField.tsx

```diff
@@ -20,12 +21,24 @@ export const TableSearchField = ({
     value = '',
     onChange,
     className,
-    placeholder,
+    placeholder: customPlaceholder,
     hasFilters,
     getSearchContext,
 }: ITableSearchFieldProps) => {
+    const ref = useRef<HTMLInputElement>();
     const { classes: styles } = useStyles();
     const [showSuggestions, setShowSuggestions] = useState(false);
+    const hotkey = useKeyboardShortcut(
+        { modifiers: ['ctrl'], key: 'k', preventDefault: true },
+        () => {
+            ref.current?.focus();
+            setShowSuggestions(true);
+        }
+    );
+    useKeyboardShortcut({ key: 'Escape' }, () => {
+        setShowSuggestions(suggestions => !suggestions);
+    });
+    const placeholder = customPlaceholder ?? `Search (${hotkey})`;
```

Comment:
Does this mean we're only appending the shortcut to the default placeholder, not custom ones from the prop? If so, assuming it would always work, I would argue it makes sense to always append it.